src/main/java/org/reaktivity/specification/kafka/internal/KafkaFunctions.java

```diff
@@ -1632,7 +1724,21 @@ public KafkaMergedDataExMatcherBuilder progress(
                     this.progressRW = new Array32FW.Builder<>(new KafkaOffsetFW.Builder(), new KafkaOffsetFW())
                                                  .wrap(new UnsafeBuffer(new byte[1024]), 0, 1024);
                 }
-                progressRW.item(i -> i.partitionId(partitionId).partitionOffset(offset));
+                progressRW.item(i -> i.partitionId(partitionId).partitionOffset(offset).build());
+                return this;
+            }
+
+            public KafkaMergedDataExMatcherBuilder progress(
```

Comment:
Please call this from the 2-parameter overloaded `progress` method, passing `DEFAULT_LATEST_OFFSET` for the `latestOffset` argument.